docs/content/strict.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Compiler Strict Mode
+kind: misc
+weight: 9
```

Comment:
This doesn't seem to have any effect on menu ordering. Is there some other place that also require updating?

Reply:
I'll have a closer look tomorrow! Not blocking this, I suppose, unless you don't agree with the current ordering.

Reply:
```
open-policy-agent/opa % git grep -C1 "kind: misc"
docs/content/comparison-to-other-systems.md-title: Comparison to Other Systems
docs/content/comparison-to-other-systems.md:kind: misc
docs/content/comparison-to-other-systems.md-weight: 16
--
docs/content/ecosystem.html-title: "Ecosystem"
docs/content/ecosystem.html:kind: misc
docs/content/ecosystem.html-weight: 10
--
docs/content/editor-and-ide-support.md-title: Editor and IDE Support
docs/content/editor-and-ide-support.md:kind: misc
docs/content/editor-and-ide-support.md-weight: 11
--
docs/content/faq.md-navtitle: FAQ
docs/content/faq.md:kind: misc
docs/content/faq.md-weight: 17
--
docs/content/oauth-oidc.md-title: OAuth2 and OpenID Connect
docs/content/oauth-oidc.md:kind: misc
docs/content/oauth-oidc.md-weight: 5
--
docs/content/schemas.md-title: Type Checking
docs/content/schemas.md:kind: misc
docs/content/schemas.md-weight: 2
--
docs/content/strict.md-title: Compiler Strict Mode
docs/content/strict.md:kind: misc
docs/content/strict.md-weight: 9
--
docs/content/wasm.md-title: WebAssembly
docs/content/wasm.md:kind: misc
docs/content/wasm.md-weight: 1
```

This seems to correspond to the ordering in the sidebar, does it not?

![image](https://user-images.githubusercontent.com/870638/149900306-d4058e28-8389-4712-974b-066107f42a15.png)


There're gaps, but that side, the weight prescribes the ordering. Or have you attempted to put it someplace else in the list? 🤔